query-engine/driver-adapters/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "private": true,
+  "name": "js",
+  "version": "0.0.2",
+  "description": "",
+  "engines": {
+    "node": ">=16.13",
+    "pnpm": ">=8.6.6 <9"
+  },
+  "license": "Apache-2.0",
+  "scripts": {
+    "build": "pnpm -r run build",
+    "lint": "pnpm -r run lint",
+    "clean": "git clean -nXd -e !query-engine/driver-adapters"
```

Comment:
Is this `clean` command still relevant?

Reply:
It is, I expect to use it from time to time, yes.